packages/augur-ui/src/modules/app/reducers/sidebar-status.ts

```diff
@@ -8,35 +8,36 @@ import {
 import { RESET_STATE } from "modules/app/actions/reset-state";
 import { MOBILE_MENU_STATES } from "modules/common-elements/constants";
 import { MARKETS } from "modules/routes/constants/views";
+import { BaseAction } from "modules/types";
 
 const DEFAULT_STATE = {
   mobileMenuState: MOBILE_MENU_STATES.CLOSED,
   currentBasePath: MARKETS,
   currentInnerNavType: null,
-  isAlertsVisible: false
+  isAlertsVisible: false,
 };
 
-export default function(sideNavStatus = DEFAULT_STATE, { type, data }) {
-  switch (type) {
+export default function(sideNavStatus = DEFAULT_STATE, action: BaseAction) {
```

Comment:
keep this as a deconstruction, we should avoid having to always do `action.data`.
`{ type, data}: BaseAction` works better